tests/ChargeTest.php

```diff
@@ -2,141 +2,90 @@
 
 namespace Stripe;
 
-class ChargeTest extends TestCase
-{
-    public function testUrls()
```

Comment:
@ob-stripe Am I correct to remove this as it's mostly tested elsewhere? Wasn't sure we tested URL-encoding properly in the lib.

Reply:
It doesn't seem like we do atm. That said I agree that `ChargeTest` is not the correct place for this. We should add an `ApiResourceTest`.